docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,321 @@ redirect_from:
+
+---
+
+### Sending and responding to actions
+
+To use features like buttons, select menus, datepickers, dialogs, and shortcuts, you’ll need to enable interactivity. Similar to events, you'll need to specify a URL for Slack to send the action (such as *user clicked a button*).
+
+Back on your app configuration page, click on **Interactivity & Shortcuts** on the left side. You'll see that there's another **Request URL** box.
+
+By default, Bolt is configured to use the same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is set up for interactivity!
+
+![Configuring a Request URL](../assets/request-url-config.png "Configuring a Request URL")
+
+Now, let's go back to your app's code and add interactivity. This will consist of two steps:
+- First, your app will send a message that contains a button.
+- Next, your app will listen to the action of a user clicking the button and respond
+
+Below, I've modified the app code we wrote in the last section to send a message with a button rather than a string:
+
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+    token=os.environ.get("SLACK_BOT_TOKEN"),
+    signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Listens to incoming messages that contain "hello"
+@app.message("hello")
+def message_hello(message, say):
+    # say() sends a message to the channel where the event was triggered
+    say(
+        blocks=[
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f"Hey there <@{message['user']}>!"
+                },
+                "accessory": {
+                    "type": "button",
+                    "text": {
+                        "type": "plain_text",
+                        "text": "Click Me"
+                    },
+                    "action_id": "button_click"
+                }
+            }
+        ],
+        text=f"Hey there <@{message['user']}>!"
+    )
+
+# Start your app
+if __name__ == "__main__":
+    app.start(port=int(os.environ.get("PORT", 3000)))
+```
+
+<!-- TODO - Review my usage of "List" (Array?) and clarification of the role of `text` -->
+The values passed to `say()` are now a list of `blocks` and a string of `text`. Blocks are the building components of a Slack message and can range from text to images to datepickers. In this case, your app will respond with a section block that includes a button as an accessory. Since we're using `blocks`, the `text` is a fallback string to display in notifications.
```

Comment:
```suggestion
The values passed to `say()` now contains an array of `blocks` and a string of `text`. Blocks are the building components of a Slack message and can range from text to images to datepickers. In this case, your app will respond with a section block that includes a button as an accessory. Since we're using `blocks`, the `text` is a fallback string to display in notifications.
```

Reply:
I'll resolve this suggestion because we reverted this paragraph back to `bolt-js`